events/guild/message.js

```diff
@@ -137,16 +146,16 @@ module.exports = async (client, message) => {
 								const lastID = doc.messageID
 								const lastTime = doc.time
 
-								const fetched = await message.channel.messages.fetch(lastID)
 								try {
-									const check = Date.now() - lastTime > 600000
+									const fetched = await message.channel.messages.fetch(lastID)
+									const check = Date.now() - lastTime > 60000
```

Comment:
60000 - Need to change this back to 600000

Reply:
Fixed in https://github.com/LukeHankey/ValenceBot/pull/32/commits/1551b530320d121d73fee4be057230d15344de70

resolve #32